example/debug/odp_debug.c

```diff
@@ -615,6 +630,7 @@ int main(int argc, char *argv[])
 		global->ipsec   = 1;
 		global->timer   = 1;
 		global->stash   = 1;
+		strcpy(global->pktio_name, "loop");
```

Comment:
If there are some arguments, but not the -i argument, pktio_name will be a zero-length string. Shouldn't this strcpy be done unconditionally after the memset?

Reply:
Packet IO is not used if -i is not listed as argument. Exception is this default case when no arguments is listed.